roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1090,7 +1090,7 @@ def set_container_facts_if_unset(facts):
         if 'ovs_image' not in facts['node']:
             facts['node']['ovs_image'] = ovs_image
 
-    if facts['common']['is_containerized']:
+    if facts['common']['is_containerized'] in ['true', 'True']:
```

Comment:
It might be better to enforce it as a boolean here: https://github.com/abutcher/openshift-ansible/blob/bz1313169-oadm/roles/openshift_facts/library/openshift_facts.py#L1419-L1426

The tricky part is that we'd need to make sure that if we've persisted the fact previously, we'll need to take that into account as well.